Assets/Scripts/TakeDamageFromPlayerProjectile.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+using System.Linq;
+
+public class TakeDamageFromPlayerProjectile : MonoBehaviour
+{
+    public float health = 25.0f;
```

Comment:
I think we can keep the health as an integer. Just leads to extra complications further down the line.

Don't know the value range of the magnitude, but having a health of something like 100 should be clear enough.

And then I think having `Mathf.Max(1, Mathf.RoundToInt(...))` should be good.